Makefile

```diff
@@ -68,7 +68,7 @@ assets:
 
 promu:
 	@GOOS=$(shell uname -s | tr A-Z a-z) \
-	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
```

Comment:
Ah right, forgot about that again.

I guess we simplify this by not trying to install promu again if it already exists. And solve the auto-update problem in promu directly. I'm just worried that all these changes need to be synced between all makefiles all the time.